controllers/board-game-review-controllers.js

```diff
@@ -57,3 +57,16 @@ exports.newComment = (req, res, next) => {
         next(err)
     });
   };
+
+  exports.editReview = (req, res, next) => {
+    const id = req.params.review_id;
+    const votes = req.body
+
+    Promise.all([checkIfReviewIdExists(id), updateReview(votes, id)])
```

Comment:
Do you need to check if the review Exists here, I think you could handle this in the update review model, which would mean you weren't contacting the database twice